addons/binding/org.openhab.binding.network/ESH-INF/thing/thing-types.xml

```diff
@@ -45,9 +46,17 @@
 				<label>Use System Ping</label>
 				<description>Uses the ping program from the system</description>
 				<default>false</default>
-			</parameter>		
+			</parameter>
+			<parameter name="arping" type="boolean">
+				<label>Use arping</label>
+				<description>Use arping instead of regular ping. Only works on Unix and arping needs to be installed. When using arping it is required to specify a network interface.</description><default>false</default>
+				</parameter><parameter name="interface" type="text">
+				<label>Network interface</label>
+				<description>When using arping, specify the network interface to use (e.g. eth0, wlan0)</description>
+				<required>false</required></parameter>
+			
 
-            <parameter name="dhcplisten" type="boolean">
+			<parameter name="dhcplisten" type="boolean">
```

Comment:
You are mixing tabs and spaces. Please follow the style guide. In the environment of openHAB, do so by using the included code formatter ([more](http://docs.openhab.org/developers/development/guidelines#a-code-style))